New Bluespace Productions Products!

AyyRobotics opened the pull request:
## About The Pull Request

Adds the mining encampment capsule, the emergency medical capsule, the security barrier capsule XL,  the *party capsule*, and the space shelter capsule, crates to buy them in, some sprites for them, some notes, and the "BSP Collapsable Shuttle."

Also adds "Bluespace Productions" as a Nanotrasen Subsidiary, only shows up in the notes.

Encampment: A 14x16 base with the amenities of standard pods, but with a custom food vendor and equipment vendor
![image](https://user-images.githubusercontent.com/60838314/82839198-94fe4500-9e9c-11ea-9625-1c7b40853147.png)

Emergency Medical: a 7x5 capsule made for situations when medical can't keep up, contains a sleeper, stasis bed (same sprite), and a nanomed.
![image](https://user-images.githubusercontent.com/60838314/82839230-ac3d3280-9e9c-11ea-88d5-b7bc6603f5b4.png)

Security Barrier: a 3x3 glass barricade for security use with laser weapons. Perfect against nukies. Windoors are public.
![image](https://user-images.githubusercontent.com/60838314/82839276-d989e080-9e9c-11ea-8a5b-e7203d31baaa.png)

Space Shelter: a (technically) 9x9 shelter with most of the amenities of a standard shelter capsule, but more spacey.
![image](https://user-images.githubusercontent.com/60838314/82839249-c414b680-9e9c-11ea-8078-367275d3ccd0.png)

Party Capsule: a 7x7 party area for when the station is in the mood to celebrate. Comes in the Party Crate.
![image](https://user-images.githubusercontent.com/60838314/83043369-cb5cd100-a010-11ea-8909-227da45d4e3f.png)

BSP Collapsable Shuttle: a small, $2500 emergency shuttle with the theme of a bluespace shelter, contains seating for around ~20 and has two sleepers, wildlife turrets at the front (space carp, I guess?), and the most important, a CAS deck in the maint area.
![image](https://user-images.githubusercontent.com/60838314/82839312-f3c3be80-9e9c-11ea-80ea-188c5eec562f.png)

## Why It's Good For The Game

Encampment:
The mining shelter capsules never really seemed like a "base" in the sense that the aux does. This remedies that by having a not-cheap not-expensiveish capsule that fits that role. Luxury and Bar aren't real bases either, and unlike them, this one focuses more on utility rather then luxury.

Emergency Medical:
We've all had those times in which medical gets overwhelmed and doesn't have enough sleepers/doctors to take care of a situation, or when medical is too far away from a threat (like blob). This helps that by providing a decently-sized, decently-stocked medical area for those doctors on the front line.

Security Barrier: Lets face it, barrier grenades are boring. Place one single barricade down and that's it. Somewhat effective but not used often. This helps that by allowing sec to deploy a hallway-sized barricade that they are able to shoot through. Combined with a riot officer or a wall, this barricade could likely hold down areas quite effectively from many threats.

Space Shelter: This one is two-fold, as it can both provide a great starting place for people to build things off-station, but can be useful in a pinch when space exploring or getting stuck floating in space.

Party: I mean, partying. Can't go wrong there can you?

BSP Collapsable Shuttle: Cheap, looks good, on the smaller side but good if there isn't going to be many people escaping that day. Nothing really special here, just another shuttle variant and those are always nice.

## Changelog
:cl:
add: Added 5 new bluespace capsules
add: Added a new shuttle
add: Adds Bluespace Productions as a Nanotrasen subsidiary
imageadd: added new capsule icons
balance: Party Crate costs 2500 due to the addition of a capsule.
/:cl:

AyyRobotics committed:
Adds lore papers

AyyRobotics committed:
Adds capsule items

AyyRobotics committed:
Adds capsule code

AyyRobotics committed:
Lets cargo purchase 3/4 capsules

AyyRobotics committed:
Adds the new shuttle code

AyyRobotics committed:
Adds map files for the capsules

AyyRobotics committed:
Adds shuttle map file

AyyRobotics committed:
Adds capsule files correctly?

AyyRobotics committed:
Updating mining p1

AyyRobotics committed:
Updating mining p2

AyyRobotics committed:
Adds the camp capsule

AyyRobotics committed:
adds mining food vendor

AyyRobotics committed:
space shelter has a unique sprite now

AyyRobotics committed:
Updating mining P3

AyyRobotics committed:
Updating mining P4

zeskorion commented:
Due to all of these blooody reinforced window buffs, maybe make the barrier capsule use normal windows

AyyRobotics commented:
Wait hold on fuck I put 2 comms consoles in the shuttle by accident

AyyRobotics committed:
Made a mistake in the shuttle

AyyRobotics committed:
fixed a thing with the shuttle

AyyRobotics commented:
Alright fixed it

AyyRobotics commented:
Damn you Travis

AyyRobotics committed:
Didn't mean to put this here

AyyRobotics committed:
Didn't mean to put this here

AyyRobotics committed:
Didn't mean to put this here

AyyRobotics committed:
Didn't mean to put this here

AyyRobotics commented:
Okay I think I fixed what got messed up, is there a way to like, restart travis or something

AyyRobotics committed:
Trying to restart Travis

PowerfulBacon commented:
How does spawning multiple shuttles capsules work?

AyyRobotics commented:
The shuttle isn't a capsule, it's just capsule themed

AyyRobotics committed:
Map Merging P1

Shoutout to ike for informing me about this mystical program

AyyRobotics committed:
Map Merging P2

AyyRobotics committed:
Map Merging P3

AyyRobotics committed:
Map Merging P4

AyyRobotics committed:
Map Merging P5 (Attempting to replace)

AyyRobotics committed:
I think this is why the error exists

AyyRobotics commented:
Alright im working on fixing the errors

AyyRobotics committed:
Fixing 1/2 of the last bug

AyyRobotics committed:
Im a fucking spaz

forgot to change one of the uniforms and instead changed the mining boots instead

AyyRobotics committed:
I am a fucking spaz Part 2

I forgot to #include the vending machine file

AyyRobotics commented:
Oh thank fuck

BettonnCZ commented:
+1 cause its cool and good

PowerfulBacon reviewed (commented):
I see a lot of tiny fans in the middle of rooms rather than in the doors

PowerfulBacon commented on _maps/shuttles/emergency_shelter.dmm:
```diff
@@ -0,0 +1,698 @@
+"ab" = (
+/obj/effect/spawner/structure/window/plastitanium,
+/turf/open/space/basic,
+/area/shuttle/escape)
```
Why is this area the escape shuttle?

AyyRobotics replied:
Oh, I think that's meant to be a passthrough, hold on

AyyRobotics replied:
Oh the windows, hold on

AyyRobotics committed:
Windows didn't have plating under them

AyyRobotics committed:
Added a first aid kit to the space shelter

AyyRobotics committed:
Medical tent didn't have a fan

AnjelicaChaykovsky commented:
You based motherfucker, i recently made a post about how cool  it would be to make more use of bluespace capsules and  it actually exists

AyyRobotics committed:
Moving the fans P1

Bacon said their main purpose was to block atmos flow and putting them in the middle of rooms wouldn't do that

AyyRobotics committed:
Moving the fans P2

AyyRobotics commented:
> You based motherfucker, i recently made a post about how cool  it would be to make more use of bluespace capsules and  it actually exists

Well ain't that convenient? There is indeed unused potential when it comes to these

AyyRobotics commented:
Fuck I forgot that that map had the old filepath, hold on

Okay fixed it

AyyRobotics committed:
Forgot that the old map had the shitty filepath

AyyRobotics committed:
7500 bro really das too much

Dropped the price of  the camp capsule from 7500 to 5000

AnjelicaChaykovsky commented:
So you didnt see my forums post where  i talked about capsules around two weeks ago? 
Damn thats a shame  i wish i could have a chat with you in that post.

AyyRobotics commented:
> So you didnt see my forums post where  i talked about capsules around two weeks ago?
> Damn thats a shame  i wish i could have a chat with you in that post.

Yeah, didn't end up seeing it, bet there were some good ideas there. At least it's not too difficult to make more!

AyyRobotics committed:
Added a sign for the medical tent

AyyRobotics committed:
Adds sign code

AyyRobotics committed:
Medical and Engineering get signs

AyyRobotics committed:
Party 1

AyyRobotics committed:
Party 2

AyyRobotics committed:
Sprite change + Party 3

AyyRobotics committed:
Party 3 + Sprites

AyyRobotics committed:
Party 4

AyyRobotics commented:
Forgot that Tweak is the wrong word for it, changed it to balance

AnjelicaChaykovsky commented:
Is this stuff going to be crate only?  
Or  is it going to be in vendors too? 
Also what is the process for adding new room to  capsule?

AyyRobotics committed:
Medical Capsule isn't as neon now

AyyRobotics commented:
> Is this stuff going to be crate only?
> Or  is it going to be in vendors too?
> Also what is the process for adding new room to  capsule?

4 of them are crate exclusive, the barricade, party, medical, and space capsules are either given their own crates or, in the case of the party crate, is included there. The mining camp capsule is gonna be in the vendor though.
To make a new room, I just make a new .dmm file in /templates, go to "shelters.dm" and put the code there, which is as easy as copying one and changing the values needed. For the item itself, I just go to "survival_pod.dm" and copy the code from another one, and replace the values needed.
Also, if your adding your own, don't worry about the "shelter_id" box in shelters.dm, just put whatever you want to call the shelter id in it. For instance, the party crate uses "shelter_theta"

Dingo-Dongler commented on code/game/objects/structures/signs/signs_departments.dm:
```diff
@@ -73,6 +73,10 @@
 
+/obj/structure/sign/departments/medbay/alt2
+	icon_state = "medicalsign"
+	desc = "An image of the Caduceus, with a small cross. Makes no sense to someone not versed in human mythology. You'll probably get help here."
```
The rod of Asclepius is the medical one. Just change it to "Makes no sense to someone ~~not~~ versed in human mythology."

AyyRobotics replied:
I originally thought that aswell, but it does seem that the Caudecus is also used as a medical symbol, even when the Rod of Asclepius is the more accurate one. But yeah the text could use an update, I'll get on that

AyyRobotics committed:
Fixing some text that looked bad

AyyRobotics commented:
Oh come on

AyyRobotics committed:
Little text update to restart the build checker

AyyRobotics commented:
After looking into this bug it seems it's byond-based, and as such I both have no idea how to solve it and have no idea if it will affect merging.

PowerfulBacon commented:
Inconsistent areas still, some are emergency shuttle for some reason, some are NOOP and some are survival pods.

AyyRobotics commented:
As-in, not the problem with my code

AyyRobotics commented:
> Inconsistent areas still, some are emergency shuttle for some reason, some are NOOP and some are survival pods.

Oh shit, a'ight I'll handle this, give me a bit

AyyRobotics committed:
Updating the sign text

AyyRobotics commented:
> Inconsistent areas still, some are emergency shuttle for some reason, some are NOOP and some are survival pods.
> 
> Oh shit, a'ight I'll handle this, give me a bit

Alright after looking, I don't see any inconsistencies. All the noops are in the correct places, the only use of the escape shuttle area is for the escape shuttle, and all the survival pod ones are in their correct places so I dunno where you see them. If you could point out the specific ones are that'd be great, thanks

PowerfulBacon commented:
Oh, I see you added another shuttle too

Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)

AyyRobotics commented:
> Oh, I see you added another shuttle too
> Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)

I thought I revised them for that, lemme check

AyyRobotics commented:
> Oh, I see you added another shuttle too
> Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)
> 
> I thought I revised them for that, lemme check

Alright I *did* revise them for that, though I missed a few spots. Gonna update that right quick. Pictures are a bit outdated

PowerfulBacon commented:
> > Oh, I see you added another shuttle too
> > Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)
> > I thought I revised them for that, lemme check
> 
> Alright I _did_ revise them for that, though I missed a few spots. Gonna update that right quick. Pictures are a bit outdated

Oh, I apologise then, I've been looking at the pictures. I'll have a long in fastdmm2 at some point.

AyyRobotics committed:
Forgot a few fans

AyyRobotics committed:
Forgot a few fans 2

AyyRobotics commented:
> Oh, I see you added another shuttle too
> Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)
> I thought I revised them for that, lemme check
> 
> Alright I did revise them for that, though I missed a few spots. Gonna update that right quick. Pictures are a bit outdated
> 
> Oh, I apologise then, I've been looking at the pictures. I'll have a long in fastdmm2 at some point.

All good, did fix what needed fixing though, so thanks for pointing that out

AyyRobotics commented:
Oh also that bug I had sorted itself out. Neat.

zeskorion reviewed (changes requested):
Not sure about the sec one using reinforced glass, especially the fucking hollow windows that take ages to deconstruct

Remove the basalt from the encampment, because it makes no sense if spawned in space

AyyRobotics commented:
> Not sure about the sec one using reinforced glass, especially the fucking hollow windows that take ages to deconstruct
> Remove the basalt from the encampment, because it makes no sense if spawned in space

I considered that when making it, however it would just leave the courtyard full of rock when it was spawned, so I decided against it so it would both be easier to access and just look better.

AyyRobotics committed:
Changing reinforced windows to normal windows

By request of Zesko

AyyRobotics committed:
Little sprite update
